src/main/java/no/nav/veilarbaktivitet/avtaltMedNav/ForhaandsorienteringDAO.java

```diff
@@ -97,6 +96,14 @@ public Forhaandsorientering getById(String id) {
         }
     }
 
+    public List<Forhaandsorientering> getById(List<String> ider) {
+        String idString = ider.stream().collect(Collectors.joining("','", "'","'"));
+        //language=sql
+        String sql = String.format("SELECT * FROM FORHAANDSORIENTERING WHERE id IN(%s)",idString);
```

Comment:
Det er et par ting her som ikke er ideelt sett fra et databaseteknisk ståsted. Det ene er at siden hver sql blir ulik, må Oracle beregne aksessplanen på nytt hver gang. Det andre er at det er en maks størrelse på 1000 på in-listen, så vi kan teoretisk sprekke på dette, selv om det kanskje er litt søkt (kanskje ikke for Aremark :)